_posts/2020-09-19-github-actions-with-faasd.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Github Actions for OpenFaas functions with faasd"
+description: "Bring CI-CD with Github Actions to your OpenFaaS functions running on a faasd appliance"
+date: 2020-09-19
+image: /images/2020-09-19-github-actions-faasd/lego.jpg
+categories:
+  - faasd
+  - containerd
+  - cicd
+  - github
+author_staff_member: utsav
+dark_background: true
+---
+
+Learn how to bring CI-CD to your OpenFaaS functions running on a faasd appliance with Github Actions
+
+## A few words about faasd
+
+OpenFaaS when came into existence gave the promise of giving users freedom to run their serverless functions anywhere, be it on a Raspberry Pi, in a data center or in the cloud. As of today OpenFaaS offers faas-netes to run OpenFaaS on Kubernetes, which is the recommended way to run OpenFaaS when running your functions in production.
```

Comment:
the opening seems to be jumbled
>OpenFaaS when came into existence gave the promise of giving users freedom 

I think you mean

> OpenFaaS was created to have the freedom to run serverless functions anywhere, be it on a Raspberry Pi, in a data center, or the cloud. This started with faas-swarm to run on Docker Swarm and faas-netes to run on Kubernetes. faas-netes is the current recommendation for running OpenFaaS in production. 
>
> Although it's not necessary ...

Reply:
Yes it definitely sounds much better.